src/interfaces/token.ts

```diff
@@ -179,7 +179,7 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
     token: Address,
     amount: Integer,
     account: Address
-  ): Promise<TransactionResponse | Boolean> {
+  ): Promise<TransactionResponse | boolean> {
```

Comment:
> Note that, the boolean Boolean is different from the lower case boolean type. The upper case Boolean is an object type whereas lower case boolean is a primitive type. It is recommended to use the primitive type boolean in your code, because, while JavaScript coerces an object to its primitive type, the TypeScript type system does not. TypeScript treats it like an object type.

Reply:
yeah don't think this was done intentionally, just a mistake, good to fix